test/unit/uk/gov/hmrc/exports/services/WCOPointerMappingServiceIntegrationTest.scala

```diff
@@ -45,10 +47,9 @@ class WCOPointerMappingServiceIntegrationTest extends WordSpec with MustMatchers
           PointerSection("declaration", PointerSectionType.FIELD),
           PointerSection("items", PointerSectionType.FIELD),
           PointerSection("1", PointerSectionType.SEQUENCE),
-          PointerSection("additionalInformation", PointerSectionType.FIELD),
-          PointerSection("items", PointerSectionType.FIELD),
+          PointerSection("documentProduced", PointerSectionType.FIELD),
           PointerSection("2", PointerSectionType.SEQUENCE),
-          PointerSection("code", PointerSectionType.FIELD)
+          PointerSection("documentIdentifier", PointerSectionType.FIELD)
```

Comment:
Key Change: For whatever reason this was passing before but shouldn't have been (as the pointer didn't exist anymore)